website/source/docs/guides/acl.html.md

```diff
@@ -247,7 +246,6 @@ with a configuration file that enables ACLs. We can use the same ACL agent token
 {
   "acl" : {
     "enabled" : true,
-    "default_policy" : "deny",
```

Comment:
I don't agree with removing this line. Even if "deny" is the default, I feel like permissions should always be made explicit to avoid mistakes.

Reply:
This is a client side example. According to [Configuring ACLs](https://www.consul.io/docs/agent/acl-system.html#configuring-acls) this setting does not apply (N/A) in client configuration.